config/initializers/countries.rb

```diff
@@ -0,0 +1,16 @@
+country_config = {
+  'IN': {
+    abbreviation: 'IN',
+    name: 'India',
+    dashboard_locale: ENV['DEFAULT_PREFERRED_DASHBOARD_LOCALE'] || 'en_IN',
```

Comment:
These values should be country specific constants, right?

Reply:
Yes. The second values in these statements are the country-specific ones. The env var just allows us to override the country-specific values _if_ we need to. I suspect that we won't need to do this any time soon.